core/build.sbt

```diff
@@ -19,6 +19,7 @@ lazy val optic =
       version := "0.1-SNAPSHOT",
       libraryDependencies ++= Seq(
         "io.github.cquiroz" %%% "scala-java-time" % "2.0.0-RC2",
```

Comment:
side note, I'm probably going to get rid of this in favor of the pluggable approach we've been using so the js side can provide an implementation of this when we need it. Hope it brings the bundle size down!